cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -233,7 +234,25 @@ public Deployment generateDeployment() {
         varList.add(buildEnvVar(ENV_VAR_KAFKA_CONNECT_METRICS_ENABLED, String.valueOf(isMetricsEnabled)));
         heapOptions(varList, 1.0, 0L);
         jvmPerformanceOptions(varList);
+        if (logConfig != null) {
+            varList.add(buildEnvVar(ENV_VAR_KAFKA_CONNECT_LOGGING, logConfig));
+        }
 
         return varList;
     }
+
+    @Override
+    protected Map getDefaultLogConfig() {
+        LinkedHashMap defaultSettings = new LinkedHashMap();
+
+        defaultSettings.put("log4j.appender.CONSOLE", "org.apache.log4j.ConsoleAppender");
+        defaultSettings.put("log4j.appender.CONSOLE.layout", "org.apache.log4j.PatternLayout");
+        defaultSettings.put("log4j.appender.CONSOLE.layout.ConversionPattern", "%d{ISO8601} %p %m (%c) [%t]%n");
+        defaultSettings.put("connect.root.logger.level", "INFO");
+        defaultSettings.put("log4j.rootLogger", "${connect.root.logger.level}, CONSOLE");
+        defaultSettings.put("log4j.logger.org.apache.zookeeper", "ERROR");
+        defaultSettings.put("log4j.logger.org.I0Itec.zkclient", "ERROR");
+        defaultSettings.put("log4j.logger.org.reflections", "ERROR");
+        return defaultSettings;
```

Comment:
the same as before cannot we use a resource file instead of hard coding ? It's the same even for TopicOperator and ZookeeperCluster I guess.